test/regressions/index.test.ts

```diff
@@ -282,6 +282,43 @@ async function main() {
       });
     });
 
+    describe('charts', () => {
+      it('should take a screenshot of the print preview', async function test() {
+        this.timeout(20000);
+
+        const route = '/docs-charts-export/PrintChart';
+        const screenshotPath = path.resolve(screenshotDir, `.${route}Print.png`);
+        await fse.ensureDir(path.dirname(screenshotPath));
+
+        // await navigateToTest(route);
+        await page.goto(`${baseUrl}${route}#no-dev`);
```

Comment:
I couldn't make this work with `navigateToTest` (client-side routing), so had to resort to `page.goto` which is a bit slower.